content/developer/reference/backend/testing.rst

```diff
@@ -540,8 +540,116 @@ business flow.  It explains a sequence of steps that should be followed.  The
 test runner will then create a Chrome headless browser, point it to the proper url
 and simulate the click and inputs, according to the scenario.
 
+Writing a test tour
+-------------------
+
+Structure
+~~~~~~~~~
+
+.. code-block:: text
+
+    your_module
+    `-- ...
+    `-- static
+        `-- tests
+            `-- tours
+                |-- your_tour.js
+    `-- tests
+        |-- __init__.py
+        |-- test_calling_the_tour.py
+    |-- __manifest__.py
+
+and ``__manifest__.py`` contains:
+
+.. code-block:: python
+
+    'assets': {
+        'web.assets_tests': [
+            'your_module/static/tests/tours/your_tour.js',
+        ],
+    },
+
+and as for any python test, ``__init__.py`` contains::
+
+    from . import test_calling_the_tour
+
+Javascript
+~~~~~~~~~~
+
+1. Setup your tour by defining it
+
+    .. code-block:: javascript
+
+        odoo.define('rental_product_configurator_tour', function (require) {
+        "use strict";
+
+        let tour = require('web_tour.tour');
+
+        tour.register('rental_product_configurator_tour', {
+            url: "/web",
+            test: true,
+        }, [tour.stepUtils.showAppsMenuItem(), {
+            // First step. Here, starting from the Apps Menu
+            trigger: '.o_app[data-menu-xmlid="sale_renting.rental_menu_root"]',
+            edition: 'enterprise'  // You don't have to precise your edition if it's in community
+        }, {
+            // Another tour step
```

Comment:
a step example where you use the extra trigger to make sure the form was correctly saved? might be interesting

Reply:
This trigger my ptsd... :laughing:

Reply:
I finally choose another traumatic experience, where I needed to see if there was a certain button instead, because it also shows that not everything require a "run" action. 2 birds...